src/components/controller/controller.ts

```diff
@@ -0,0 +1,31 @@
+import AppLoader from './appLoader';
+import { ArticleResponse, SourceResponse } from '../../types';
+
+class AppController extends AppLoader {
+    getSources(callback: (data: SourceResponse) => void): void {
+        super.getResp<SourceResponse>(
+            {
+                endpoint: 'sources',
+            },
+            callback
+        );
+    }
+
+    getNews(callback: (data: ArticleResponse) => void): void {
+        const dropdown = document.querySelector('.sources-dropdown') as HTMLSelectElement;
+        console.log(dropdown);
```

Comment:
While using console.log is a great dev tool, it should extremely rarely, if ever, be present in production applications. Please clean up console.log's across the project (except the ones present in it initially)